keras/utils/data_utils.py

```diff
@@ -364,18 +364,30 @@ def on_epoch_end(self):
         """
         pass
 
+# Global variables to be shared across processes
+shared_sequence = None
+manager = multiprocessing.Manager()
+shared_dict = manager.dict()
```

Comment:
Please use all caps for global variables, and use a `_` prefix to indicate they are private.